// @x402/extensions/dist/esm/bazaar/index.mjs
import {
  BAZAAR,
  bazaarResourceServerExtension,
  declareDiscoveryExtension,
  extractDiscoveryInfo,
  extractDiscoveryInfoFromExtension,
  extractDiscoveryInfoV1,
  extractResourceMetadataV1,
  isBodyExtensionConfig,
  isDiscoverableV1,
  isMcpExtensionConfig,
  isQueryExtensionConfig,
  isValidIconUrl,
  isValidRouteTemplate,
  isValidServiceName,
  sanitizeResourceServiceMetadata,
  sanitizeTags,
  validateAndExtract,
  validateDiscoveryExtension,
  validateRouteTemplate,
  withBazaar
} from "../chunk-RWD3IB5N.mjs";
export {
  BAZAAR,
  bazaarResourceServerExtension,
  declareDiscoveryExtension,
  extractDiscoveryInfo,
  extractDiscoveryInfoFromExtension,
  extractDiscoveryInfoV1,
  extractResourceMetadataV1,
  isBodyExtensionConfig,
  isDiscoverableV1,
  isMcpExtensionConfig,
  isQueryExtensionConfig,
  isValidIconUrl,
  isValidRouteTemplate,
  isValidServiceName,
  sanitizeResourceServiceMetadata,
  sanitizeTags,
  validateAndExtract,
  validateDiscoveryExtension,
  validateRouteTemplate,
  withBazaar
};
